styles/twitch/catppuccin.user.css

```diff
@@ -247,14 +247,11 @@
       --color-text-button-disabled: @subtext0;
     }
 
-    &,
-    [class*="ScTokenOverrideCSSVars"] {
```

Comment:
Pretty sure this will break stuff? Last I checked this was needed or else some elements weren't properly themed... Iirc it was somethign about @'s in stream titles (find a stream that has an @ in the title, hover over the @ to get a card to popup)?

Reply:
I checked it now, and @'s in titles and it's popups are properly themed.